pages/windows/cipher.md

```diff
@@ -1,13 +1,17 @@
 # cipher
 
-> Encrypt or decrypt files on NTFS drives.
+> Encrypt, decrypt files on NTFS drives, or show their encryption information.
```

Comment:
Not sure about this
```suggestion
> Encrypt or decrypt files on NTFS volumes, or show their encryption information.
```

Reply:
Or based on the man page
```suggestion
> Displays or alters the encryption of directories and files on NTFS volumes.
```

Reply:
The second one is a good general description, i guess.
Thanks for your suggestion!